whelktool/scripts/cleanups/2019/09/remove-muskogean-language-from-music-posts/script.groovy

```diff
@@ -0,0 +1,47 @@
+/*
+ * This removes https://id.kb.se/language/mus from all posts that are not really Muskogean
+ *
+ * 2435 posts with https://id.kb.se/language/mus
+ * 2405 of those have type "NotatedMusic" or "Music"
+ * Manual inspection of the titles of the 30 remaining gave one post with actual language Muskogean
+ *
+ * This has the problem of removing Muskogean from all "Music" or "NotatedMusic" that is actually in Muskogean...
+ *
+ * ID list generated with
+ * curl '10.50.16.190:9200/libris_stg/_search?type=bib&q=(Muskogeanska)&size=3000&filter_path=hits.total,hits.hits._id&pretty=true' | grep _id | tr '"' ' ' | awk '{print $3}' > muskogeanska_ids.txt
```

Comment:
Good note! Let's use use-cases like this to refine our public (find) API and examples thereof as well. Right now we can do:
```
curl -s 'https://libris-stg.kb.se/find.json?q=Muskogeanska&@type=Instance&_limit=3000' |
    jq -r '.items[] |.["@id"]' |
    sed 's|.*/\([^/]\+\)#it|\1|'
```
(Which is workable but a rather heavy result bulk. We probably 1) want to add the `_lens`-parameter to `/find` to get just chips for each item, and 2) add a simple tsv-view based on that (I've got a half-baked one in a stash...).)